src/main/java/com/toptrumps/cli/CommandLineUI.java

```diff
@@ -78,8 +79,9 @@ public void requestNumberOfPlayers() {
         }
     }
 
-    public void showPlayerCard() {
-
+    public void showPlayerCard(Card c) {
+        System.out.println("You drew \'" + c.getDescription() + "\':");
```

Comment:
This probably needs rephrasing but we can decide that later

Reply:
I copied the CLI example output. It absolutely can be changed later, but it's just a place holder.